frontend/src/metabase/admin/settings/components/CloudPanel/MigrationStart.tsx

```diff
@@ -0,0 +1,82 @@
+import { t } from "ttag";
+
+import { getPlan } from "metabase/common/utils/plan";
+import { useToggle } from "metabase/hooks/use-toggle";
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+import { Modal, Box, Button, Text, List, Flex } from "metabase/ui";
+
+interface MigrationStartProps {
+  startNewMigration: () => void;
+  isStarting: boolean;
+}
+
+export const MigrationStart = ({
+  startNewMigration,
+  isStarting,
+}: MigrationStartProps) => {
+  const [isModalOpen, { turnOn: openModal, turnOff: closeModal }] =
+    useToggle(false);
+
+  const plan = useSelector(state =>
+    getPlan(getSetting(state, "token-features")),
+  );
+  const isProSelfHosted = plan === "pro-self-hosted";
+
+  return (
+    <>
+      <Box mt="1rem">
+        <Text size="md">
+          {t`It only takes a few clicks to migrate this instance to Metabase Cloud.`}
+          {isProSelfHosted && " "}
+          {isProSelfHosted
+            ? t`There is no additional cost for your Pro account.`
+            : ""}
+        </Text>
+
+        <Text size="md">{t`You will get:`}</Text>
+
+        <List size="md">
+          <List.Item>{t`Automatic upgrades`}</List.Item>
+          <List.Item>{t`Freedom from having to manage your own server`}</List.Item>
+          <List.Item>{t`Preconfigured SMTP Server`}</List.Item>
```

Comment:
```suggestion
          <List.Item>{t`A preconfigured SMTP server`}</List.Item>
```

"You will get preconfigured SMTP server" doesn't feel right to me. I think there should be an "a" in there.